tokio/src/net/tcp/stream.rs

```diff
@@ -184,6 +184,56 @@ impl TcpStream {
         Ok(TcpStream { io })
     }
 
+    /// Turn a [`tokio::net::TcpStream`] into a [`std::net::TcpStream`].
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::error::Error;
+    /// use std::io::Read;
+    /// use tokio::net::TcpListener;
+    /// # use tokio::net::TcpStream;
+    /// # use tokio::io::AsyncWriteExt;
+    /// # use tokio::time::Duration;
+    ///
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), Box<dyn Error>> {
+    /// #   tokio::spawn(async {
+    /// #       tokio::time::sleep(Duration::from_millis(50)).await;
+    /// #       let mut stream: TcpStream = TcpStream::connect("127.0.0.1:8080").await.unwrap();
+    /// #       let _ = stream.write(b"Hello world!").await;
+    /// #   });
+    ///     let mut data = [0 as u8; 50];
+    ///     let listener = TcpListener::bind("127.0.0.1:8080").await?;
+    ///     let (tokio_tcp_stream, _) = listener.accept().await?;
+    ///
+    ///     let mut std_tcp_stream = tokio_tcp_stream.into_std()?;
+    /// #   tokio::time::sleep(Duration::from_millis(100)).await;
+    ///     let size = std_tcp_stream.read(&mut data)?;
+    /// #   assert_eq!("Hello world!", std::str::from_utf8(&data[0..size])?);
+    /// #   Ok(())
```

Comment:
Without this, it wont compile if the user pastes it into the playground.
```suggestion
    ///     Ok(())
```